### PR TITLE
plugin for IMAP_SMTP injection check

### PR DESCRIPTION
**Pull request by ph1ne4s**
This plugin is to verify the capacity to inject arbitrary IMAP/SMTP commands into the mail servers, due to input data not being properly sanitized.

## Description
The `command-injection-tester` python tool allows for straightforward testing of email servers for the STARTTLS command injection vulnerability in SMTP, POP3, and IMAP.

## Reviewers
@viyatb 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [ x] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -22,4 +22,5 @@ tornado==5.1.1
 bcrypt==3.2.0
 PyJWT==1.7.1
 bs4==0.0.1
-pyotp==2.6.0
\ No newline at end of file
+pyotp==2.6.0
+command-injection-tester
```
Comment (viyatb) [inline requirements/base.txt]: can you put a version here?

Comment (ph1ne4s) [reply requirements/base.txt]: done.

**Comment by viyatb** [comment]
can you fix the merge conflict? I think I merged your other PR first.

**Comment by ph1ne4s** [comment]
> can you fix the merge conflict? I think I merged your other PR first.

done

File: owtf/plugins/web/external/Testing_WSDL@OWTF-WS-002.py [hunk 2]
```diff
@@ -7,4 +7,4 @@
 def run(PluginInfo):
     resource = get_resources("ExternalWebServices")
     Content = plugin_helper.resource_linklist("Online Resources", resource)
-    return Content
+    return Content
```
Comment (viyatb) [inline owtf/plugins/web/external/Testing_WSDL@OWTF-WS-002.py]: Can you fix the extra space?

File: owtf/data/conf/resources.cfg [hunk 3]
```diff
@@ -1189,9 +1189,8 @@ SnmpProbeMethods_____SNMP Probing using _____@@@TOOL_METASPLOIT_DIR@@@/msfconsol
 BruteSnmpProbeMethods_____SNMP Probing using _____@@@TOOL_METASPLOIT_DIR@@@/msfconsole -q -x "use auxiliary/scanner/snmp/snmp_login; set RHOSTS @@@host_ip@@@; set RPORT @@@SNMP_PORT_NUMBER@@@; run; exit"
 PassiveOpenS3Buckets_____Grayhat Warfare (Public Buckets)_____https://buckets.grayhatwarfare.com/results/@@@host_name@@@
 SubdomainTakeover_____Subdomain Takeover_____sh @@@FRAMEWORK_DIR@@@/scripts/subdomain_takeover.sh @@@host_name@@@
+IMAP_SMTPinjection_____IMAP_SMTP Injection_____ command-injection-tester --imap --pop3 --smtp @@@host_name@@@
```
Comment (viyatb) [inline owtf/data/conf/resources.cfg]: can you fix the space between `____` and `command-injection-tester`?

**Review by viyatb — CHANGES_REQUESTED** [review]

File: owtf/plugins/web/external/Testing_WSDL@OWTF-WS-002.py (hunk 2 above)
Comment (ph1ne4s) [reply owtf/plugins/web/external/Testing_WSDL@OWTF-WS-002.py]: done

File: owtf/data/conf/resources.cfg (hunk 3 above)
Comment (ph1ne4s) [reply owtf/data/conf/resources.cfg]: done

**Pull request merged**